change/@microsoft-teams-js-6d28f626-b164-4d85-ad16-098e9b7491dd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Adding search capability to use global search box in the current app in Outlook",
```

Comment:
As per the [contribution guidelines](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball), please ensure this commit message is using past tense and enclose types (like `search`) in backtick quotes ("`"), e.g.,

"Added \`search\` capability to use the global search box in the current application in Outlook."